skribenten-web/frontend/src/Brevredigering/LetterEditor/editor.css

```diff
@@ -1,11 +1,11 @@
 .editor {
     margin-top: var(--a-spacing-6);
-    width: 758px;
+    width: 393px;
```

Comment:
Denne må revereteres og således endre testene. Hjalp å gjøre bredde mindre for å teste edge caser